pages/global-profile/views/index.jsx

```diff
@@ -11,22 +11,46 @@ import Modules from '../../profile/read/views/modules';
 import RelatedTasks from '../../task/list/views/related-tasks';
 import AsruRoles from '../components/asru-roles';
 
-const formatDate = (date, format) => (date ? dateFormatter(date, format) : '-');
+const selector = ({
+  model,
+  static: {
+    allowedActions,
+    asruUser,
+    isOwnProfile
+  }
+}) => ({
+  model,
+  allowedActions,
+  asruUser,
+  isOwnProfile
+});
 
-class Index extends React.Component {
+const formatDate = (date, format) => (date ? dateFormatter(date, format) : '-');
 
-  render () {
+export default function Index({ dedupe, AsruRolesComponent, children }) {
+  const { model, allowedActions, asruUser, isOwnProfile } = useSelector(selector, shallowEqual);
+  const hasEstablishments = !!(model.establishments || []).length;
 
-    const model = this.props.model;
-    const isOwnProfile = this.props.isOwnProfile;
-    const hasEstablishments = !!(model.establishments || []).length;
+  const showEmail = asruUser || model.asruLicensing || isOwnProfile;
+  const showDob = (asruUser || isOwnProfile) && model.dob;
```

Comment:
I think we probably need to filter this on the server-side and then simply render here if it is defined.

Otherwise the data will be accessible through devtools.